pkg/manager/manager.go

```diff
@@ -365,6 +365,10 @@ func NewManager(opt Opt) (*Manager, error) {
 	return mgr, nil
 }
 
+func (m *Manager) GetCacheDir() string {
```

Comment:
There is a similar method `CacheDir`

Reply:
emm... Yes, but this method is belong to another package `cache`. I will change `GetCacheDir` to `CacheDir`.